docs/ref/catalog-formats.rst

```diff
@@ -53,28 +53,32 @@ This is how the regular PO format exports plurals:
 
 With `po-gettext`, plural messages are exported in the following way, depending on wheter an explicit ID is set:
 
-.. code-block:: po
+1. Message **with custom ID "my_message"** that is pluralized on property "someCount".
+
+   Notice that 'msgid_plural' was generad by appending a '_plural' suffix.
```

Comment:
```suggestion
   Notice that 'msgid_plural' was generated by appending a '_plural' suffix.
```

Reply:
@andrii-bodnar Thanks, I only moved the sections and didn't check the text contents 🙂